Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -620,6 +742,21 @@ public void NewItem(string name, string location, ContentItemTemplate template)
             _actionStack.Add(action);
         }
 
+        public void NewFolder(string name, string location)
+        {
+            string folder = Path.Combine(location, name);
+
+            if (!Path.IsPathRooted(folder))
+                folder = _project.Location + Path.DirectorySeparatorChar + folder;
+
+            if (!Directory.Exists(folder))
+                Directory.CreateDirectory(folder);
```

Comment:
We might want to protect against any exception thrown from directory creation and fire up an error dialog.